sql/Pg-upgrade-2.8.32-2.8.33.sql

```diff
@@ -0,0 +1,48 @@
+ALTER TABLE IF EXISTS bank_account
+    ADD PRIMARY KEY (id);
+
+CREATE TABLE IF NOT EXISTS bank_account_balance (
+    id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+    bank_account_id INT REFERENCES bank_account (id) NOT NULL,
+    balance_type TEXT NOT NULL,
+    amount DOUBLE PRECISION NOT NULL,
+    credit_debit TEXT NOT NULL,
+    date DATE NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS bank_file_import (
+    id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+    bank_account_id INT REFERENCES bank_account (id) NOT NULL,
+    total_amount DOUBLE PRECISION NOT NULL,
+    transactions_count INT NOT NULL,
+    imported_when DATE NOT NULL,
+    source_type TEXT NOT NULL,
+    is_manual BOOLEAN NOT NULL,
+    filename TEXT
+);
```

Comment:
I would also save the entire contents of the file so that you can check later whether everything went correctly.

Reply:
file_content added